src/main/java/de/lukweb/justmail/imap/responses/ImapResponse.java

```diff
@@ -14,7 +14,7 @@
     }
 
     public String create(String tag, String description) {
-        return tag + " " + text + " " + description;
+        return tag + " " + text + " " + description + "\r\n";
```

Comment:
I would override the send method of the Sessions within the ImapSession class, but you don't have to do this. Just update your fork and use the new method.